linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords_etc_shadow/tests/an_empty_pass_present.fail.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```

Comment:
Please add `passwd` to the list of required packages to ensure that the `passwd` command used in this test scenario will be present inside the test container used in the CI.
```suggestion
# packages = passwd
```